Bookr/Bookr/Books.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Books.swift
+//  Bookr
+//
+//  Created by Aaron on 8/22/19.
+//  Copyright © 2019 John Kouris. All rights reserved.
+//
+
+import Foundation
+
+struct Books {
```

Comment:
We might want to make the final attribute a variable if we are going to be adding/deleting reviews